doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-section.adoc

```diff
@@ -16,29 +16,128 @@ After login, users may upload dashboard files exported from {CENTRAL}
+dashbuilder/perspectives/page1/perspective_layout.plugin
+dashbuilder/perspectives/page2/perspective_layout
+dashbuilder/perspectives/page2/perspective_layout.plugin
+dashbuilder/perspectives/readme.md
+dashbuilder/navigation/navigation/navtree.json
+dashbuilder/navigation/readme.md
+VERSION
+----
+
+
+If you want to export customized user created dashbuilder data as a ZIP file, do the following tasks:
+
+... On the *Dashbuilder Data Transfer* page, click *Custom export*.
+... In *Export Wizard* panel, select the datasets that you want to include in the ZIP file and click *Next*.
+... Select the pages that you want to include in the ZIP file and click *Next*.
++
+The *Export Wizard* panel validates the selected datasets and pages. The summary of the datasets and pages is available on the panel.
++
+NOTE: Navigation is always included in the exported `ZIP` file.
+
+... Click *Download* if your export is ready.
++
+An `export.zip` file containing customized dashbuilder data is downloaded.
++
+NOTE: You must select the pages and the datasets associated with those pages in the *Export Wizard* panel. You must select at least one page. If you fail to select both datasets and pages, an export error is generated and you can not download the `export.zip` file.
+
+... Click *Finish*.
+
+--
+
+== Importing Dashbuilder data
+
+You can import Dashbuilder data to {CENTRAL} from a ZIP file if the archive is structured in the same way as the following example:
+
+[source]
+----
+dashbuilder/datasets/definitions/dataset-example1.dset
+dashbuilder/datasets/definitions/dataset-example2.dset
+dashbuilder/datasets/readme.md
+dashbuilder/perspectives/page1/perspective_layout
+dashbuilder/perspectives/page1/perspective_layout.plugin
+dashbuilder/perspectives/page2/perspective_layout
+dashbuilder/perspectives/page2/perspective_layout.plugin
+dashbuilder/perspectives/readme.md
+dashbuilder/navigation/navigation/navtree.json
+dashbuilder/navigation/readme.md
+VERSION
+----
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
++
+WARNING: You should only import Dashbuilder data to a clean installation of {PRODUCT} in order to avoid overwriting data on an existing system.
+
+. On the *Dashbuilder Data Transfer* page, click the *Choose File* icon.
+
+. Navigate to the ZIP file you want to import and select the file.
+
+. Click the *Upload* icon.
+
+. Click *Import*.
+
+== Deploying dashboards from {CENTRAL} on dashbuilder runtime
+
+You can automatically deploy the dashboards from {CENTRAl} on dashbuilder runtime. {CENTRAl} is linked to dashbuilder runtime with the help of gradual export feature.
+
+.Prerequisites
+
+* Dasbuilder runtime is configured on the system.
+* You must set the `dashbuilder.runtime.multi`, `dashbuilder.runtime.location`, and `dashbuilder.export.dir` system properties to `true` in the `standalone.xml` file to enable the feature.
```

Comment:
This is the same text as in one of the previous files. Could you please fix this?

Reply:
Changes the system properties description.